src/audio/google/google_rtc_audio_processing.c

```diff
@@ -385,9 +390,21 @@ static int google_rtc_audio_processing_init(struct processing_module *mod)
 	md->private = cd;
 
 #if CONFIG_IPC_MAJOR_4
-	struct module_config *cfg = &md->cfg;
+	const struct ipc4_base_module_extended_cfg *base_cfg = md->cfg.init_data;
+	struct ipc4_input_pin_format reference_fmt, output_fmt;
+	const size_t size = sizeof(struct ipc4_input_pin_format);
+
+	memcpy_s(&cd->config.base_cfg, sizeof(struct ipc4_base_module_cfg),
+		&base_cfg->base_cfg, sizeof(struct ipc4_base_module_cfg));
```

Comment:
any concern you don't want to use below code?
cd->config.base_cfg = base_cfg->base_cfg;

afraid inside pointers?

Reply:
yeah that would work as well. fixed now